aldryn_people/cms_plugins.py

```diff
@@ -20,32 +20,59 @@ class PeoplePlugin(CMSPluginBase):
     name = _('People list')
     model = models.PeoplePlugin
 
-    def group_people(self, people, language):
+    fieldsets = (
+        (None, {
+            'fields': (
+                'style',
+            ),
+        }),
+        (_('People'), {
+            'description': _('Select and arrange specific people, or leave '
+                             'blank to use all.'),
+            'fields': (
+                'people',
+            )
+        }),
+        (_('Options'), {
+            'fields': (
+                ('group_by_group', 'show_ungrouped', ),
+                'show_links',
+                'show_vcard',
+            )
+        })
+    )
+
+    def group_people(self, people):
         groups = defaultdict(list)
 
         for people in people:
             for group in people.groups.all():
                 groups[group].append(people)
 
-        # Python/Django bug ?
+        # Fixes a template resolution-related issue. See:
+        # http://stackoverflow.com/questions/4764110/django-template-cant-loop-defaultdict  # noqa
         groups.default_factory = None
         return groups
 
     def render(self, context, instance, placeholder):
         people = instance.get_selected_people()
+        if not people:
+            people = models.Person.objects.all()
         self.render_template = self.TEMPLATE_NAME % instance.style
 
         context['instance'] = instance
         context['people'] = people
 
-        if (models.Group.objects.filter(people__in=people).exists() and
-                instance.group_by_group):
-            context['people_groups'] = self.group_people(
-                people, instance.language)
-            context['group_less_people'] = people.filter(groups__isnull=True)
+        if instance.group_by_group:
+            context['people_groups'] = self.group_people(people)
+            if instance.show_ungrouped is True:
```

Comment:
Why the identification? If it's True the expression already has the value you want.